doc/usage/extensions/coverage.rst

```diff
@@ -22,6 +22,14 @@ should check:
 
 .. confval:: coverage_ignore_classes
 
+.. confval:: coverage_ignore_py
```

Comment:
`ignore_py` is a bit ambiguous to me. What about `coverage_ignore_pyobject` instead?

Reply:
Switched to `coverage_ignore_pyobjects` (plural to keep it coherent with the other 3 Python options).